### ISSUE TITLE
Could not parse `flambe.platform.ManifestBuilder.use(build/.cache/air/assets)`

### ISSUE THREAD

**Issue by accidentalrebel** [aduros/flambe#309]
This is the continuation of the problem I've outlined [here](https://groups.google.com/forum/#!topic/flambe/1ON9O0rjBI4). The problem [was fixed](https://github.com/HaxeFoundation/haxe/issues/3674#event-207417574) by using a nightly version of Haxe. However, a new problem has appeared and it seems to be a Flambe issue.

> --macro:1: character 0 : Could not parse `flambe.platform.ManifestBuilder.use(build/.cache/air/assets)`
> --macro:1: character 0 : Unexpected .

The code I am using is below:

> haxe -main hyper_app.Main -lib flambe -cp src -dce full --no-traces --macro flambe.platform.ManifestBuilder.use("build/.cache/air/assets") --flash-strict -swf-header 640:480:60:000000 -D native_trace -swf-lib-extern build/.cache/air/admob_all_in_one_6.12.2.ane.swc -swf-lib-extern build/.cache/air/AirFacebook.ane.swc -swf-lib-extern build/.cache/air/Flurry.ane.swc -swf-lib libs/Flurry.swc -swf-version 14 -D air -D ios -D no-flash-override -swf build/.cache/air/main-ios.swf

Thinking that it might be caused by the quotes, I changed:

> flambe.platform.ManifestBuilder.use("build/.cache/air/assets")

to 

> flambe.platform.ManifestBuilder.use('build/.cache/air/assets')

With the above change, no more error appears but unfortunately, I still can't deploy to IOS. Anyone have any ideas?

============================================

**Some additional notes:**
- I am using the nightly build for Flambe.
- To make it work with an extension I am using, I am using "-swf-version 14" instead of the default "-swf-version 11.7". Not sure if this affects anything but I thought that it might help out.

============================================

**Comment by accidentalrebel** [comment]
Found the cause of the problem. I was using Air 15.0.0 and it wan't working well with the IOS deployment of the latest version of Flambe.

To fix: I downgraded to 14.0.0 and removed the "useLegacyAOT=true" in the "index.js" file of the flambe source.

It seems to be working fine aside from a warning that appears:
> ld: warning: CPU_SUBTYPE_ARM_ALL subtype is deprecated: C:\\Development\\Tools\\
AirSDK\\14.0\\lib\\aot/lib/libNoDebugger1.arm-air.a(avmplusDebuggerStubs.cpp.o)

I'll mark this as resolved.

Thanks everyone~!

**Comment by markknol** [comment]
Thanks for the update on this!

**Comment by camaech** [comment]
Is this resolved in the current nightly flambe build or do I need to do as @accidentalrebel is stating?

**Comment by markknol** [comment]
You need to fix it manually atm.

**Comment by kreso22** [comment]
@accidentalrebel I can't seem to find how to remove "useLegacyAOT=true" from "index.js". Could you tell me where I can find it?

Thanks!!
Kreso

**Comment by aduros** [comment]
You need to grab the source (use the hacking instructions in the readme), then change remove the line containing -useLegacyAOT in command/index.js.

This is only an interim fix. It will make your app build and run again... but Apple will reject your appstore submission if you use this. I'm working on a proper fix that will allow appstore submissions.

**Comment by kreso22** [comment]
Thank you for the answer Bruno. I know flambe is a passion project, but I am just curious if you have any guestimate for this feature?

Thanks again,
Kreso

**Comment by aduros** [comment]
Not sure yet, but it's high on my priorities. I would guestimate a week or so.

**Comment by accidentalrebel** [comment]
@kreso22  Yeah, sorry if I was vague. As @aduros  has mentioned, you need to get the Flambe source and edit from there.

> but Apple will reject your appstore submission if you use this. I'm working on a proper fix that will allow appstore submissions.

Ah, so it seems that mine is just a temporary fix. Reopening the issue as it is not fully resolved.

**Comment by markknol** [comment]
Any new fixes ready for this issue?

**Comment by apands** [comment]
Hi, is there still no fixes? I have a potential deal including iOS version, so it'd be cool to hear something about it :)

**Comment by benwager** [comment]
Hi, Just wanted to give a quick +1 on this issue. Any word on a fix? Timescale? Thanks
